.github/actions/pytest/action.yml

```diff
@@ -0,0 +1,27 @@
+name: pytest
+description: installs package, runs pytest and creates coverage report.
+
+inputs:
+  backend:
+    description: which database backend to use
+    required: false
+  args:
+    description: arguments that are passed to pytest, e.g. "-k filter" or "tests/integration"
+    required: false
```

Comment:
I've added a new action that runs our tests based on an argument
The goal was to save repetitive code here, but the end result is just a few lines shorter on the main `ci.yml` file